portainer/portainer.go

```diff
@@ -0,0 +1,181 @@
+package portainer
```

Comment:
Why is this its own package?

Reply:
Fixed 89a8bb0864090344bdd9b4460a6bd9785c47f045